### ISSUE TITLE
run error and questions about data format

### ISSUE THREAD

**Issue by BraveY** [thu-pacman/GeminiGraph#7]
Hi, i'm studying the brilliant system.
After deploying the system ,i got error 'Segmentation fault (core dumped)' when i tried to run it . Is there any suggestion for this error ?
More else, where can i got the file 'twitter-2010.binedgelist'? If it is possibile , Would you please offer some test files in this project so that users could run this system more conveniently without worrying the data format.
Thanks

**Comment by coolerzxw** [comment]
Hi. Can your provide more detailed information? You may use gdb to locate the detailed position where the segmentation fault happens.
For the twitter-2010 data, you can first go to the WebGraph Dataset page and then write a program to extract the edges and write them to an output file in binary format.

**Comment by SummerLitchy** [comment]
@coolerzxw So do you mean that based on current project, the only way of reading the graph is read from a binary format from a regulated exact format?
If is, I think the data reading part could be more flexible. If you need the part I think I am willing to help. (I am currently trying this on a practical problem before this I used GraphX)
qiu hui fu 3q! ╮(╯▽╰)╭

**Comment by coolerzxw** [comment]
@SummerLitchy Yes the current implementation can only read from binary edge list files. It would be great if you can enhance the input/output implementation!

**Comment by jackhouchina** [comment]
hi, how to make twitter-2010 data ,it's download twitter-2010.graph and write it to binary format?
